README.md

```diff
@@ -202,12 +202,14 @@ For local development, running the ```mit``` harvester is recommended.
 
 Note: harvester-name is the same as the defined harvester "short name".
 
-Invove a harvester a certain number of days back with the ```--days``` argument. For example, to run a harvester 5 days in the past, run:
+Invoke a harvester a certain number of days back with the ```--days``` argument. For example, to run a harvester 5 days in the past, run:
 
 ```bash
 $ invoke harvester harvester-name --days=5
 ```
 
+Writing a harvester for inclusion with scrAPI?  [autooai](https://github.com/erinspace/autooai) is a utility that makes this easy for most cases.
```

Comment:
Hi @felliott! A great addition! One suggestion, can you specify that this is for writing a harvester for sources that have their metadata in OAI-PMH format?